tests/unit/checkpoint/test_zero_optimizer.py

```diff
@@ -576,3 +577,33 @@ def test_save_tensor_clone(self, tmpdir, zero_stage, use_cpu_device):
         torch.save(clone_state_dict, clone_ckpt_file)
 
         compare_state_dicts(torch.load(ref_ckpt_file), torch.load(clone_ckpt_file))
+
+     
+class TestZeRONonDistributed(DistributedTest):
+    world_size = 1
+    init_distributed = False
+
+    def test_chmod_exception_handling(self, monkeypatch):
+        
+        config_dict = {"train_batch_size": 1, "zero_optimization": {"stage": 0}}
```

Comment:
ZeRO is activated by `stage >=1`, so can you clarify why using `stage = 0`?

Reply:
Yea for the test, access to the method was all that was really needed. 
So I just used minimal settings, if you prefer I can change it to `stage=1`

Reply:
It would be better to parametrize with `stage = [2,3]` to  provide more coverage. Thanks!

Reply:
Sounds good pushed that with `[1,2,3]`